plugins/peermanager/manager.go

```diff
@@ -55,30 +46,26 @@ func (service *PeerManagerService) attachPeers(peer string) {
 	log.Info("added peer, peer manager plugin", "added", peer)
 }
 
-func (service *PeerManagerService) chainIdResolver() (string, error) {
-
-	var chainID string
-	err := service.client.Call(&chainID, "eth_chainId")
-	if err != nil {
-		return "", err
-	}
+func chainIdResolver(id int64) string {
 	var result string
-	switch chainID {
-	case "0x1":
+	switch id {
+	case 1:
 		result = "mainnet"
-	case "0x3d":
+	case 61:
 		result = "etc"
-	case "0x4268":
+	case 17000:
 		result = "holesky"
-	case "0xaa36a7":
+	case 11155111:
 		result = "sepolia"
-	case "0x89":
+	case 137:
 		result = "polygon"
-	case "0x13881":
+	case 80001:
 		result = "mumbai"
+	case 80002:
+		result = "amoy"
 	default:
 		log.Warn("unknown chain, chainID could not be resolved, peer manager plugin")
-		result = chainID
+		result = "unknownChain"
```

Comment:
We need this to at least reflect the chainid, otherwise if we're running multiple different unsupported chains they'll publish to the same topic and try to peer with eachother.

Let's set this to

```
result = fmt.Sprintf("%x", id)
```